.github/pull_request_template.md

```diff
@@ -1,15 +1,52 @@
-Describe your changes here.
+_Authors: Please fill out this form carefully and completely._
 
-Closes # (type the issue number after # if applicable; otherwise remove this line)
+_Reviewers: By approving this Pull Request you are approving the code change, as well as its deployment and mitigation plans._
+_Please read this description carefully. If you feel there is anything unclear or missing, please ask for updates._
+
+#### List the issues that this change affects.
+<!--Every code change must address _at least 1_ issue. Fixes a bug, completes a task, every change
+      should have a corresponding issue listed here. If one does not already exist, create one. -->
+      
+Closes # (type the GitHub issue number after #)
+
+### What are you trying to accomplish?
+<!-- Provide a description of the changes. -->
 
 ### Screenshots
+<!-- Provide before/after screenshots, videos, or graphs for any visual changes; otherwise, remove this section -->
+
+#### Risk Assessment
+  <!-- Please select from one of the following and detail why this level was chosen -->
+
+- [ ] **Low risk** the change is small, highly observable, and easily rolled back.
+- [ ] **Medium risk** changes that are isolated, reduced in scope or could impact few users. The change will not impact library availability.
+- [ ] **High risk** changes are those that could impact customers and SLOs, low or no test coverage, low observability, or slow to rollback.
+
+### What approach did you choose and why?
+<!-- This section is a place for you to describe your thought process in making these changes.
+     List any tradeoffs you made to take on or pay down tech debt.
+     Identify any work you did to mitigate risk.
+     Describe any alternative approaches you considered and why you discarded them. -->
+
+### Anything you want to highlight for special attention from reviewers?
+<!-- This is your chance to identify remaining risks and confess any uncertainties you may have about the correctness of the changes.
+     Highlight anything on which you would like a second (or third) opinion.
+     Keep in mind how many component uses cases may be affected by your changes when assessing risk. -->
```

Comment:
I think this section could be framed as "testing & reviewing" in order to provide a way for authors to provide reviewers with specific context into the PR and how it could be tested/reviewed

```suggestion
## Testing & Reviewing

<!-- Describe any specific details to help reviewers test or review this Pull Request -->
```

Reply:
Updated, see diff: https://github.com/primer/react/pull/3344/files#diff-b2496e80299b8c3150b1944450bd81c622e04e13d15c411d291db0927d75fd6bR21